### ISSUE TITLE
how to downdown the pretrained model

### ISSUE THREAD

**Issue by JasonQAQ** [VitoLin21/Rodent-Skull-Stripping#1]
Your Google Drive link seems to be inaccessible,how to request the access?

**Comment by VitoLin21** [comment]
Sorry, the Google Drive link is temporarily unavailable as we have not yet organized the files for the annotated brain masks. However, you can download the pre-trained model from the following link: [MEGA](https://mega.nz/file/KltFGQZK#ISdwNTb7bDafT9zWeTwHrdNbQMYZYka7NBxm-UVb300)

**Comment by VitoLin21** [comment]
Google drive is accessible now🙂.

**Comment by JasonQAQ** [comment]
thanks a lot
